### DISCUSSION TITLE
PASSWORD_AUTHENTICATION_FEATURE

### DISCUSSION THREAD

**Discussion by davidmoshal** [AllYourBot/hostedgpt#462] [General]
Hi, this looks like an awesome project, however am unable to log into my hosted server on Render. 

I can log into my Render account, but when I click the hostedgpt webservice and get the login page, it fails with my Render password.

So, I tried 2 things:

a) It's possible that I need another password to log into the hosted server, though there is no documentation about how to set this up.

b) I tried to set this: PASSWORD_AUTHENTICATION_FEATURE
But there is no documentation on where to set it!
Is it set on the webserve instance, the database instance, or at the level of Env groups one wonders?

thanks

**Comment by keithschacht** [comment]
Hi @davidmoshal, you should simply sign up. Did you try that? Sorry I’ve made this too confusing! You’re right, setting the FEATURE flags isn’t super clear. I was thinking that most people would never need to change these, but I should document that better.

By default, password authentication is turned on. And if you’re seeing the login screen, then that confirms that part is enabled.

It should be that you go to your domain, it takes you to the login screen but you need to sign up first since this app was just created. After you sign up, you can login with that newly created account.
